modules/cnv-configuring-guest-memory-overcommitment.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-managing-guest-memory.adoc
+
+[id="cnv-configuring-guest-memory-overcommitment_{context}"]
+= Configuring guest memory overcommitment
+
+If your virtual workload requires more memory than available, you can
+use memory overcommitment to allocate all or most of the host’s memory
+to your virtual machine instances. Enabling memory overcommitment means
+you can maximize resources that are normally reserved for the host.
+
+For example, if the host has 32 GB RAM, you can leverage memory
+overcommitment to fit 8 VMs with 4 GB RAM each. This works under the
+assumption that the VMs will not use all of their memory at the same
+time.
+
+// .Prerequisites
+
+// *
+
+.Procedure
+
+* To explicitly tell the VMI that it has more memory available than what
```

Comment:
VMI? If it's virtual machine instance, define the initialism in the intro.

maybe s/what has been requested/was